pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -412,7 +442,9 @@ public enum PGProperty {
 
   /**
    * Hostname of the PostgreSQL server (may be specified directly in the JDBC URL).
+   * @deprecated Use property HOST instead
    */
+  @Deprecated
   PG_HOST(
       "PGHOST",
```

Comment:
What happens to the code that still uses `PGHOST` and `PGPORT` properties? Will it continue to work? Where's the corresponding test case?

Reply:
test cases and fixes added: https://github.com/pgjdbc/pgjdbc/pull/2646/commits/e9676f259dad716e29d1cee12e809fab9de50351